airbyte-integrations/connectors/source-lever-hiring/source_lever_hiring/schemas.py

```diff
@@ -0,0 +1,191 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+
+from typing import Any, List, MutableMapping, Optional, Type
+
+import pydantic
+from pydantic import BaseModel
+from pydantic.typing import resolve_annotations
+
+
+class AllOptional(pydantic.main.ModelMetaclass):
+    def __new__(self, name, bases, namespaces, **kwargs):
+        """
+        Iterate through fields and wrap then with typing.Optional type.
+        """
+        annotations = resolve_annotations(namespaces.get("__annotations__", {}), namespaces.get("__module__", None))
+        for base in bases:
+            annotations = {**annotations, **getattr(base, "__annotations__", {})}
+        for field in annotations:
+            if not field.startswith("__"):
+                annotations[field] = Optional[annotations[field]]
+        namespaces["__annotations__"] = annotations
+        return super().__new__(self, name, bases, namespaces, **kwargs)
+
+
+class BaseSchemaModel(BaseModel, metaclass=AllOptional):
+    class Config:
+        arbitrary_types_allowed = True
+
+        @classmethod
+        def schema_extra(cls, schema: MutableMapping[str, Any], model: Type["BaseModel"]) -> None:
+            schema.pop("title", None)
+            schema.pop("description", None)
+            for name, prop in schema.get("properties", {}).items():
+                prop.pop("title", None)
```

Comment:
what is the value of removing these? can you add a comment? 

I'm wondering if we shouldn't have `title` and `description` set where possible since it may be a useful documentation

Reply:
The `title `field will "copy" the field name in `Camel Case`, and the `desctiption `field will remain empty. Globally, we may not remove these when generating schemas, but they do not carry absolutely any payload, so we remove them.

Reply:
makes sense. Can you add a comment with this context?